pkg/nfs/deploy/kubernetes/csi-attacher-nfsplugin.yaml

```diff
@@ -0,0 +1,64 @@
+# This YAML file contains attacher & csi driver API objects that are necessary
```

Comment:
These files look like they are a duplicate of `deploy/kubernetes/*`

Reply:
ahh .. missed to delete them. will fix it.

Reply:
Done !